Makefile

```diff
@@ -88,6 +89,19 @@ release-image:
 	@echo "Remember you need to push your image before calling make deploy"
 	@echo "    make push"
 
+debug-image: ## Build a docker image with the broker binary started in debug mode
+	docker run --rm -v ${BUILD_DIR}:/tmp/artifact docker.io/philipgough/dlv:centos cp /go/bin/dlv /tmp/artifact
```

Comment:
I'm happy to maintain this image. This is a workaround to support those who cannot do muti stage builds

Reply:
Is there a reason you can't `go get` it?